### PR TITLE
Code ready for final review before merging with master branch

### PR DESCRIPTION
**Pull request by Osama-Tahboub1**
Code ready for final review before merging with master branch

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+.DS_Store
+.idea
```
Comment (charliecog) [inline .gitignore]: whats the .idea file? if its not being used then just delete it

File: about.html [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+        <h1>About Me</h1>
+    </header>
+
+    <section class="container aboutMe">
+        <article>
+            <p>
+                I am currently attending Mayden Academy to become a full stack developer using HTML, CSS, Java Script, PHP, SQL and Node.js.
+            </p>
+
```
Comment (charliecog) [inline about.html]: unnecessary whitespace

File: about.html [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    <footer class="container">
+        <div class="contactIcon">
+            <span>Contact me</span>
+            <a href="mailto:o.tahboob2000@gmail.com?Subject=Portfolio%20Query"></a>
+            <img src="assets/contactWhite.png" alt="Email me icon">
+        </div>
+
+        <div class="contactIconSmall">
+            <span>Contact me</span>
```
Comment (charliecog) [inline about.html]: not displaying on page - remove if not using

File: index.html [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+<footer class="container">
+    <div class="contactIcon">
+        <span>Contact me</span>
+        <a href="mailto:o.tahboob2000@gmail.com?Subject=Portfolio%20Query"></a>
+        <img src="assets/contactWhite.png" alt="Email me icon">
+    </div>
+
+    <div class="contactIconSmall">
+        <span>Contact me</span>
```
Comment (charliecog) [inline index.html]: not displayed

File: portfolio.html [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+                    <li><a href="about.html">About</a></li>
+                    <li><a href="#">Portfolio</a></li>
+                </ul>
+            </nav>
+
+            <h1>My Portfolio</h1>
+        </header>
+
+        <main>
+            <div class="col3 col2">
```
Comment (charliecog) [inline portfolio.html]: identify with col is happening based on size eg: col3 and col2-sm

File: portfolio.html [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+                <h2><a href="/sites/academy-php7/html/Responsive site project/index.html">Pilot Shop</a></h2>
```
Comment (charliecog) [inline portfolio.html]: need to change all links from absolute to relative within your project folders as these links wont work on the server

File: portfolio.html [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+
+            <div class="col3 col2">
+                <h2><a href="/sites/academy-php7/html/Responsive site project/index.html">Pilot Shop</a></h2>
+                <img src="assets/PilotShop.png" alt="Portfolio sample image">
+            </div>
+        </main>
+
+        <footer class="container">
+            <div class="contactIcon">
+                <span>Contact me</span>
```
Comment (charliecog) [inline portfolio.html]: not being displayed

File: styles.css [hunk 8]
```diff
@@ -0,0 +1,253 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    margin: 0;
+    background: url(assets/background1.jpeg);
```
Comment (charliecog) [inline styles.css]: missing " " around the url

File: styles.css [hunk 9]
```diff
@@ -0,0 +1,253 @@
...
+    background-repeat: no-repeat;
+    background-size: cover;
+    font-family: Serif;
```
Comment (charliecog) [inline styles.css]: lowercase s

File: styles.css [hunk 10]
```diff
@@ -0,0 +1,253 @@
...
+    top: 0px;
+    right: 0px;
+    font-size: 3rem;
+    background-color: rgba(128, 128, 128, 0.5);
+}
+
+header h1 {
+    position: fixed;
+    top: 40px;
+    left: 0px;
```
Comment (charliecog) [inline styles.css]: dont need px for anything with a 0

File: styles.css [hunk 11]
```diff
@@ -0,0 +1,253 @@
...
+
+.container {
+    margin: 0 auto;
+    overflow: auto;
+    width: 100%;
+}
+
+nav {
+    position:fixed;
+    top: 0px;
```
Comment (charliecog) [inline styles.css]: dont need px for anything with a 0

File: styles.css [hunk 12]
```diff
@@ -0,0 +1,253 @@
...
+    display: none;
+}
+
+.aboutMe {
+    position: absolute;
+    top: 30%;
+}
+
+section article {
+    padding:2rem 5rem;
```
Comment (charliecog) [inline styles.css]: space after the :

File: styles.css [hunk 13]
```diff
@@ -0,0 +1,253 @@
...
+    .contactIcon {
+        display: block;
+        position: fixed;
+        top: 53%;
+        right: 5px;
+        background-color: transparent;
+    }
+
+    section article {
+        padding:1rem 1rem;
```
Comment (charliecog) [inline styles.css]: space after :

File: index.html (hunk 4 above)
Comment (Osama-Tahboub1) [reply index.html]: displays in IE9 as a replacement to the icon, since link to generate email does not work in IE9 with icon, so it becomes visible in IE9 and the icon disappears.

File: .gitignore (hunk 1 above)
Comment (Osama-Tahboub1) [reply .gitignore]: Not sure, I didn't create it, so was concerned it was a system file, if its not important I can delete it.

File: about.html (hunk 3 above)
Comment (Osama-Tahboub1) [reply about.html]: Used in IE9 degradation.

File: about.html (hunk 2 above)
Comment (Osama-Tahboub1) [reply about.html]: I have amended this.

File: portfolio.html (hunk 7 above)
Comment (Osama-Tahboub1) [reply portfolio.html]: Used in IE9 degradation.

File: styles.css (hunk 8 above)
Comment (Osama-Tahboub1) [reply styles.css]: This is done.

File: styles.css (hunk 9 above)
Comment (Osama-Tahboub1) [reply styles.css]: This is done, just a note that Php storm default value uses capital S.

File: styles.css (hunk 11 above)
Comment (Osama-Tahboub1) [reply styles.css]: This is fixed.

File: styles.css (hunk 10 above)
Comment (Osama-Tahboub1) [reply styles.css]: This is fixed.

File: styles.css (hunk 12 above)
Comment (Osama-Tahboub1) [reply styles.css]: This is done.

File: styles.css (hunk 13 above)
Comment (Osama-Tahboub1) [reply styles.css]: This is done.

File: .gitignore (hunk 1 above)
Comment (Osama-Tahboub1) [reply .gitignore]: Seems to be a php storm file: 
https://rider-support.jetbrains.com/hc/en-us/articles/207097529-What-is-the-idea-folder-

File: portfolio.html (hunk 5 above)
Comment (Osama-Tahboub1) [reply portfolio.html]: Changed .col2 to .col2Sm, does that suffice?

File: portfolio.html (hunk 6 above)
Comment (Osama-Tahboub1) [reply portfolio.html]: As far as I can see these links are positioned relatively from ".col3 a", the only link positioned absolutely is the contact icon, where the link is positioned according to the icon, will these need to be changed? this was the only thing that had to be fixed in IE9, so might need to check it with browser stack again if this needs changing.

File: portfolio.html [hunk 14]
```diff
@@ -0,0 +1,69 @@
...
+                    <li><a href="#">Portfolio</a></li>
+                </ul>
+            </nav>
+
+            <h1>My Portfolio</h1>
+        </header>
+
+        <main>
+            <div class="col3 col2Sm">
+                <h2><a href="/sites/academy-php7/html/Responsive site project/index.html">Pilot Shop</a></h2>
```
Comment (charliecog) [inline portfolio.html]: change these to the live links where appropriate and all others to #

Comment (Osama-Tahboub1) [reply portfolio.html]: Changed two to live links ,and the others were changed to #.

**Comment by charliecog** [comment]
approved
